content/docs/gto/dvc.md

```diff
@@ -0,0 +1,5 @@
+## Working with large artifacts
+
+...Git-lfs, DVC and what else is there...
+
+...(I'll remove this page before merging this PR)...
```

Comment:
Remove incomplete files? I don't think we want to publish this.

Reply:
Oh I see it's not in the nav. I'd still remove it from the PR as a best practice. Wasn't that what the `gto-docs-update` branch was for? (temporary hold incomplete/unsure docs)

Reply:
Let's not overoptimize. It's not indexed and we'll rewrite it anyway in the next weeks.